### ISSUE TITLE
i18n for starnames.dat

### ISSUE THREAD

**Issue by levinli303** [CelestiaProject/Celestia#993]
We should add i18n for starnames.dat

**Comment by levinli303** [comment]
> Main name can be easily translated but aliases require additional work because both english and localized names are shown:
> ![image](https://user-images.githubusercontent.com/1612688/117839786-37693100-b284-11eb-8930-002d64320777.png)

I felt we should only show the localized names for any name. if the localized names is not widely recognized in that language, then probably that just should not be translated in the first place.

**Comment by 375gnu** [comment]
I agree. But the problem is how the name database implemented. I would like to store only english names and translate them when we show them. This will also fulfill the longstanding request to switch languages in runtime.

**Comment by levinli303** [comment]
https://github.com/CelestiaProject/Celestia/pull/996 should solve searching? I stored nameIndex for localized name separately (although still loaded at launch time so no runtime switch), and allowed completion to search both indices when true is passed for `i18n`.

**Comment by 375gnu** [comment]
@levinli303 please give me a few days, i'm currently busy. then i'll test your proposed changes.
